### DISCUSSION TITLE
Grail6 Java higher version

### DISCUSSION THREAD

**Discussion by deepali-bharankar-ocl** [apache/grails-core#13909] [General]
As per docs of Grail6: Starting from Grails 6, Java 11 serves as the baseline requirement for the framework.
What is the higher version of Java does Grails 6 supports like Java 17 or higher than that?

**Comment by jdaugherty** [comment]
Grails 6 runs fine with Java 17.  I am not sure Grails 6 supports 21 due to some old dependencies & its gradle version.  

Grails 7 supports definitely supports Java 21 and it may support 23 in its current state.

**Comment by matrei** [comment]
> What is the higher version of Java does Grails 6 supports like Java 17 or higher than that?

@deepali-bharankar-ocl Do you refer to building or running a Grails 6 app?

**Reply by deepali-bharankar-ocl** [reply]
Hi @matrei , I am referring to Running a Grails 6 app.

**Reply by matrei** [reply]
@deepali-bharankar-ocl Then the sky is the limit :rocket:
